android/src/main/java/com/swmansion/reanimated/nodes/SetNode.java

```diff
@@ -10,8 +13,8 @@
 
   public SetNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
     super(nodeID, config, nodesManager);
-    mWhatNodeID = config.getInt("what");
-    mValueNodeID = config.getInt("value");
+    mWhatNodeID = MapUtils.getInt(config, "what", "'what' argument passed to set node might be of wrong type. NodeID: " + nodeID);
+    mValueNodeID = MapUtils.getInt(config, "value", "'value' argument passed to set node might be of wrong type. NodeID: " + nodeID);
```

Comment:
I think we may skip these internal names in favor of sth more descriptive

Reply:
by internal names, you do mean what, and value? We can definitely do this then.

Reply:
Check this out right now, I used argument order 🙂